leverageapi/api.py

```diff
@@ -107,6 +107,7 @@ def races():
     response_str = json.dumps(resp, sort_keys=False, default=dthandler)
     response = make_response(response_str, 200)
     response.headers['Content-Type'] = 'application/json'
+    # response.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
why

Reply:
Well, the api wouldn't work for me locally otherwise and this was the quickest fix I could muster.

Reply:
should be fixed by #66